plugin/src/main/kotlin/io/github/gmazzo/android/test/aggregation/AndroidTestCoverageAggregationPlugin.kt

```diff
@@ -146,12 +153,10 @@ abstract class AndroidTestCoverageAggregationPlugin : Plugin<Project> {
 
         // Jacoco does not supports multiple versions of the same class (when merging jar of different variants)
         // So we create a unified classes dir, doing the best effort keeping the first of each variant
-        val allVariantsJars = objects.listProperty<RegularFile>()
-        val allVariantsDirs = objects.listProperty<Directory>()
         val allVariantsClassesForCoverageReport by tasks.registering(Sync::class) {
-            from(allVariantsDirs)
             into(provider { temporaryDir })
             duplicatesStrategy = DuplicatesStrategy.INCLUDE // in case of duplicated classes
+            include("**/*.class")
```

Comment:
Now required, as the `PRE_JACOCO_TRANSFORMED_CLASSES` also includes `R.jar` from its dependencies